float32 heading
float32 speed
float32 depth
bool heading_enabled
bool speed_enabled
bool depth_enabled
